src/main/java/it/gov/pagopa/pu/debtpositions/service/create/debtposition/CreateDebtPositionServiceImpl.java

```diff
@@ -72,4 +81,13 @@ private void generateIuv(DebtPositionDTO debtPositionDTO, Boolean pagopaPayment,
         });
     }
   }
+
+  private void invokeWorkflow(DebtPositionDTO debtPositionDTO, Boolean pagopaPayment, String accessToken, Boolean massive) {
+    WorkflowCreatedDTO workflow = debtPositionSyncService.invokeWorkFlow(debtPositionDTO, accessToken, pagopaPayment, massive);
+    if (workflow != null) {
+      log.info("Workflow created with id {}", workflow.getWorkflowId());
+    } else {
+      log.warn("Workflow creation failed for Debt Position id {}, origin {} and pagopaPayment {}: received null response", debtPositionDTO.getDebtPositionId(), debtPositionDTO.getDebtPositionOrigin(), pagopaPayment);
+    }
```

Comment:
just log as info...
GPD when massive or no pagopa payments will not invoke any workflow...